browser/containers/App.tsx

```diff
@@ -104,6 +116,23 @@ class App extends React.Component<Props, State> {
         { generalError != null ? this.mapError(generalError) : '' }
 
         <div className="container-fluid mt-4">
+          <div className="row">
+            <div className="mx-auto col-lg-10">
+              <Switch>
+                <Route exact path="/" component={Metrics} />
+                <Route exact path="/employee" component={Employee} />
+                <Route exact path="/list" component={List} />
+                <Route exact path="/admin" component={Admin} />
+                <Route path="/cla/:project_id" component={EditCLA} />
+                <Route exact path="/contribute" component={Contributions} />
+                <Route path="/approvals/:contrib_id" component={Approvals} />
+                <Route path="/contribution/:contrib_id" component={EditContribution} />
+                <Route exact path="/metrics" component={Metrics} />
+                <Route exact path="/contribute/link" component={GithubLinkUpdater} />
+              </Switch>
+            </div>
+          </div>
+
```

Comment:
With this added, you can probably delete the children section below here

Reply:
Interesting. I want to talk more with you about this as I am not really happy not fully understand the changes to react-router.